homeassistant/components/plugwise/sensor.py

```diff
@@ -0,0 +1,411 @@
+"""Plugwise Sensor component for Home Assistant."""
+
+import logging
+
+from homeassistant.const import (
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_POWER,
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    ENERGY_KILO_WATT_HOUR,
+    ENERGY_WATT_HOUR,
+    POWER_WATT,
+    PRESSURE_BAR,
+    TEMP_CELSIUS,
+    UNIT_PERCENTAGE,
+    VOLUME_CUBIC_METERS,
+)
+from homeassistant.helpers.entity import Entity
+
+from . import SmileGateway
+from .const import (
+    COOL_ICON,
+    DEVICE_CLASS_GAS,
+    DEVICE_CLASS_VALVE,
```

Comment:
Only the available device classes should be used.  If we want to add new device classes, an architecture issue is required https://github.com/home-assistant/architecture/issues/new

For now since we are trying to get this merged before beta, please take them out.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes
https://developers.home-assistant.io/docs/core/entity/binary-sensor#available-device-classes

Reply:
Changed accordingly - will file seperately